app/presentation/components/slider.py

```diff
@@ -0,0 +1,21 @@
+from PyQt5.QtCore import Qt
+from PyQt5.QtWidgets import QVBoxLayout, QSlider, QLabel
+from app.consts.presentation_consts import SLIDER_MAX, SLIDER_MIN, SLIDER_TIC
+
+
+class Slider(QVBoxLayout):
+    def __init__(self):
+        super().__init__()
+        self.print_scale_slider: QSlider = QSlider(Qt.Horizontal)
+        self.print_scale_slider_label: QLabel = QLabel("Scale 1")
```

Comment:
I think that we can get rid of that text field, as right now it doesn't say anything to the user